omod/src/main/resources/config.xml

```diff
@@ -34,6 +34,7 @@
 		<require_module version="${groovyVersion}">org.openmrs.module.groovy</require_module>
 		<require_module version="${uiframeworkVersion}">org.openmrs.module.uiframework</require_module>
 		<require_module version="${kenyaemrOrderentryVersion}">org.openmrs.module.kenyaemrorderentry</require_module>
+		<require_module version="${registrationCoreVersion}">org.openmrs.module.registrationcore</require_module>
```

Comment:
@moshonk I suggest you move this to aware of modules tag as opposed to required module.